api/envoy/config/cluster/dynamic_forward_proxy/v2alpha/cluster.proto

```diff
@@ -13,6 +13,10 @@ import "validate/validate.proto";
 
 // [#protodoc-title: Dynamic forward proxy cluster configuration]
 
+// [#not-implemented-hide:] Not configuration. Workaround for C++ issue with stripping proto files.
+message ClusterDummy {
```

Comment:
I might be wrong but I don't think you need any of this as there is no service here. The protos will be compiled in when the code is used

Reply:
Hm yea looks like you're right. Removed the dummy proto but kept the validation assert around because that seems useful. LMK if you think we should remove/change it